gertie/Sources/Gertie/Semver.swift

```diff
@@ -148,9 +148,9 @@ extension Semver: Codable {
 }
 
 extension Semver: LosslessStringConvertible {
-  private nonisolated(unsafe) static let semverRegexPattern =
```

Comment:
these were never really unsafe, but the apple packages weren't correctly annotated as `Sendable`